.github/workflows/release_ruleset.yaml

```diff
@@ -0,0 +1,45 @@
+# Reusable workflow that can be referenced by repositories in their .github/workflows/release.yaml.
+# See example usage in https://github.com/bazel-contrib/rules-template/blob/main/.github/workflows/release.yaml
```

Comment:
requirements:
- .github/workflows/release_prep.sh
- .github/workflows/ci.bazelrc

Reply:
Also `.bazelrc` technically

Reply:
in theory we could even check for those files and give a nice error but no time today :)

Reply:
PTAL